openfisca_us/parameters/gov/states/ny/tax/income/supplemental/recapture_base/head_of_household.yaml

```diff
@@ -0,0 +1,37 @@
+description: Recapture base for NY State supplemental tax for head-of-household filers.
+brackets:
+  - threshold:
+      2022-01-01: 0
+    amount:
+      2022-01-01: 0
+  - threshold:
+      2022-01-01: 107_650
+    amount:
+      2022-01-01: 0
+  - threshold:
+      2022-01-01: 269_300
+    amount:
+      2022-01-01: 787
+  - threshold:
+      2022-01-01: 1_616_450
+    amount:
+      2022-01-01: 3_076
+  - threshold:
+      2022-01-01: 5_000_000
+    amount:
+      2022-01-01: 48_337
+  - threshold:
+      2022-01-01: 25_000_000
+    amount:
+      2022-01-01: 80_837
+
+metadata:
+  threshold_unit: currency-USD
```

Comment:
do you need `type: single_amount`?

Reply:
It's automatically inferred by OpenFisca because of the `amount` usage, but I agree explicit is better so added.